frontend_lib/src/component/Lists/ListItemWrapper/ListItemWrapper.jsx

```diff
@@ -26,13 +26,13 @@ ListItemWrapper.propTypes = {
   contentType: PropTypes.object,
   isLast: PropTypes.bool,
   read: PropTypes.bool,
-  id: PropTypes.number
+  id: PropTypes.string
 }
 
 ListItemWrapper.defaultProps = {
   label: '',
   customClass: '',
   isLast: false,
   read: false,
-  id: 0
+  id: undefined
```

Comment:
Isn't it undefined by default?

Reply:
I wanted to make it explicit so people don't think this was forgotten, but I can remove it if you think it's better.

Reply:
No we can keep it as it is.